pyclustree/_clustree.py

```diff
@@ -365,6 +418,43 @@ def clustree(
                 bbox={"boxstyle": "round", "facecolor": facecolor[i], "edgecolor": "black"},
             )
 
+    if score_clustering is not None:
+        score_to_name: dict[ClusteringMetricsType, str] = {
+            "silhouette": "Silhouette score",
+            "calinski_harabasz": "Calinski and Harabasz score",
+            "davies_bouldin": "Davies-Bouldin score",
+        }
+        x_min = ax.get_xlim()[0]
+        x_max = ax.get_xlim()[1]
+        y_positions_levels = np.linspace(
+            ax.get_ylim()[1], ax.get_ylim()[1] - len(cluster_keys) * 1.0, len(cluster_keys)
+        )
+
+        ax.text(
+            x_max,
+            y_positions_levels[0] + 0.5,
+            s=score_to_name[score_clustering],
+            fontsize=10,
+            color="black",
+            ha="right",
+            va="center",
+        )
```

Comment:
The text in the example does not seem to be aligned with the clustering labels on the right side and the text size seems to differ